build/creator/IRedeemer.sol

```diff
@@ -11,5 +11,5 @@ interface IRedeemer {
 
     function authRedeem(uint8, address, uint256, address, address, uint256) external returns (uint256);
     function markets(uint8, address, uint256) external view returns (Market memory market);
-    function getExchangeRate(uint8, address) external view returns (uint256);
```

Comment:
we don't historically use `getFoo` anywhere so i changed this to match existing patterns